src/Juvix/Core/HR/Extend.hs

```diff
@@ -0,0 +1,24 @@
+module Juvix.Core.HR.Extend where
+
+import qualified Juvix.Core.IR.Types.Base
+import qualified Juvix.Core.IR.Types.Base as IR
+import Juvix.Library
+
+extTerm =
+  \_primTy _primVal ->
+    IR.defaultExtTerm
+      { IR.nameLam = "Lam0",
+        IR.typeLam = Just [[t|Symbol|]],
+        IR.namePi = "Pi0",
+        IR.typePi = Just [[t|Symbol|]],
+        IR.nameLet = "Let0",
+        IR.typeLet = Just [[t|Symbol|]]
+      }
```

Comment:
The default extension should already handle this for you, no!?

Reply:
I don't quite follow, this is `HR`, it adds names for variables (instead of de Bruijin indices in IR)